Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/UserController.cs

```diff
@@ -20,19 +19,18 @@ namespace Promact.Oauth.Server.Controllers
     public class UserController : BaseController
     {
         #region "Private Variable(s)"
-        private readonly ILogger _logger;
```

Comment:
Why have you removed logger? logger is useful for logging messages. Please incorporate logger properly.